src/en/anchira/src/eu/kanade/tachiyomi/extension/en/anchira/Anchira.kt

```diff
@@ -193,26 +202,42 @@ class Anchira : HttpSource(), ConfigurableSource {
 
     // Details
 
-    override fun mangaDetailsRequest(manga: SManga) =
-        GET("$libraryUrl/${getPathFromUrl(manga.url)}", headers)
+    override fun mangaDetailsRequest(manga: SManga): Request {
+        return if (manga.url.startsWith("?")) {
+            GET(libraryUrl + manga.url, headers)
+        } else {
+            GET("$libraryUrl/${getPathFromUrl(manga.url)}", headers)
+        }
+    }
 
     override fun mangaDetailsParse(response: Response): SManga {
-        val data = json.decodeFromString<Entry>(response.body.string())
+        return if (response.request.url.pathSegments.count() == libraryUrl.toHttpUrl().pathSegments.count()) {
+            val manga = latestUpdatesParse(response).mangas.first()
+            val query = response.request.url.queryParameter("s")
+            manga.apply {
+                url = "?${response.request.url.query}"
+                description = "Bundled from $query"
+                title = "[Bundle] ${manga.title}"
```

Comment:
possible to remove the chapter number from the title? or maybe use the title from "title:<>" in the query

[something like this](https://github.com/AwkwardPeak7/tachiyomi-extensions/blob/ksk/src/en/koushoku/src/eu/kanade/tachiyomi/extension/en/koushoku/Koushoku.kt#L187-L191)

Reply:
It's possible since it's already done for chapter titles, but would need to either duplicate that couple lines of code or make it it's own function since it's part of `createChapter`

Reply:
Other than duplicating the RegEx it wasn't as bad as I initially thought :)